test/test_metrics/baseclass.py

```diff
@@ -16,14 +22,54 @@ def setUp(self):
 				"native_metrics": "false",
 			}
 		})
-		self.App = asab.Application(args=[], modules=[asab.metrics.Module])
-		self.MetricsService = self.App.get_service('asab.MetricsService')
+		self.App = asab.Application(args=[], modules=[])
+		self.MetricsService = MockMetricsService(self.App, "asab.MetricsService")
 		self.MetricsService.clear()
 		self.MetricsService.Tags['host'] = "mockedhost.com"
 		self.MockedLoop = MockedLoop()
 
+	def tearDown(self):
+		asab.abc.singleton.Singleton.delete(self.App.__class__)
+		self.App = None
+		root_logger = logging.getLogger()
+		root_logger.handlers = []
+
+
 
 class MockedLoop(object):
 	def time(self):
 		return 123.45
 
+
+class MockApplication(object):
```

Comment:
Remove use of `.Loop` from the`DutyCycle`, replace by `.App` ... so that it is unified with the rest of ASAB.

Reply:
Yes, please change that.
Duty cycle is used only in BSPump, so please coordinate with @PremyslCerny and do the change in the separate MR.